### DISCUSSION TITLE
Outreachy Winter 23 Applicant Introduction - Jeevanshi

### DISCUSSION THREAD

**Discussion by Femme-js** [ModECI/MDF#338] [General]
Hi Everyone!

I am Jeevanshi, a recent engineering graduate. I have with working in the domain of AI/ML for the last 2 years. I am excited to contribute to this project. Looking forward to learning from you all.

Thanks,
Jeevanshi

**Comment by pgleeson** [comment]
Hi @Femme-js! Please have a look at https://github.com/ModECI/MDF/discussions/346 for an introduction to the project and some suggested tasks.
